src/appleseed/renderer/modeling/light/directionallight.cpp

```diff
@@ -135,7 +135,7 @@ namespace
                 light_transform,
                 s,
                 m_scene_center,
-                m_scene_radius,
+                0,
```

Comment:
Here, since you have a target point, you can compute `position`, `outgoing` and `probability` (and `value` but this one doesn't change) in a much simpler manner.